labelbox/orm/db_object.py

```diff
@@ -44,8 +45,11 @@ def __init__(self, client, field_values):
         self._set_field_values(field_values)
 
         for relationship in self.relationships():
+            value = field_values[
```

Comment:
can you do `field_values.get(relationship.name)` instead?

Reply:
It should throw a KeyError if `precompute = True` and the key does not exist. I could break this out into 
```
value =   field_values.get(relationship.name)
if relationship.precompute and value is None:
    raise KeyError(f"Expected field  values for {relationship.name}"
```
Does that make more sense?

Reply:
I think it might, this statement is a little hard to grok

Reply:
ok, I'll update it